discovery/seata-discovery-core/src/main/java/io/seata/discovery/loadbalance/LoadBalanceFactory.java

```diff
@@ -24,12 +28,23 @@
  */
 public class LoadBalanceFactory {
 
+    /**
+     * The constant LOAD_BALANCE.
+     */
+    private static final String LOAD_BALANCE = FILE_ROOT_REGISTRY + FILE_CONFIG_SPLIT_CHAR + "loadBalance";
+
+    /**
+     * The load balance.
+     */
+    private static final LoadBalance LOAD_BALANCE_INSTANCE = EnhancedServiceLoader.load(LoadBalance.class,
```

Comment:
If the configuration file does not have this configuration item, a default value is required. Compatible with previous versions

Reply:
done